src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/jpa/repository/specifications/PersonSpecification.kt

```diff
@@ -8,7 +8,7 @@ import java.time.LocalDate
 
 object PersonSpecification {
 
-  const val SEARCH_VERSION = "1.1"
+  const val SEARCH_VERSION = "1.2"
```

Comment:
Can we put in a graph in AppInsights showing average number of matches against search version?

Reply:
Would this be worth it as we'd have to search further than 24h to get that data.

Reply:
I think it would be very useful. If memory serves, you can set the time range of an individual tile to be different to the dashboard. At the moment it won't tell us much but hopefully once we start getting the numbers down it will come in handy